server/urls.py

```diff
@@ -8,6 +8,9 @@
 admin.autodiscover()
 
 urlpatterns = [
+    # user urls
+    url(r'^users/(?P<name>[a-zA-Z0-9]+)', server.views.user, name='user'),
```

Comment:
Can you remove the `users/` part here per earlier discussion?

It occurs to me that you might need to move this down a bit so we match the other patterns first.

Reply:
Apologies, misunderstood the previous comment. Will do.